### ISSUE TITLE
Save screenshot

### ISSUE THREAD

**Issue by fernandojsg** [aframevr/a-painter#5]

**Comment by Utopiah** [comment]
Should be fixed using Aframe master new screenshot component, still have to be integrated to the menu.

**Comment by Utopiah** [comment]
To take a regular (projection) screenshot, use the keyboard shortcut (<ctrl> + <alt> +s).

To take a 360° (equirectangular) screenshot, use the keyboard shortcut (<ctrl> + <alt> + + <shift> +s).

https://aframe.io/docs/master/components/screenshot.html

**Comment by Utopiah** [comment]
Can we close this?

**Comment by dmarcos** [comment]
Do we still want it in the menu?

**Comment by Utopiah** [comment]
True, would be nicer for newcomers. It means adding an option but also doing the screenshot component differently because as of now it brings up a new opening/save window that isn't visible in VR.

**Comment by dmarcos** [comment]
This is related https://github.com/aframevr/a-painter/pull/179
